utils/reflector/src/main/java/org/robolectric/util/reflector/UnsafeAccess.java

```diff
@@ -95,7 +97,7 @@ public <T> Class<?> defineClass(Class<T> iClass, String reflectorClassName, byte
         // return privateLookup.defineClass(bytecode);
         return (Class<?>) defineClassMethod.invoke(privateLookup, bytecode);
       } catch (IllegalAccessException | InvocationTargetException e) {
-        throw new AssertionError(e);
+        throw new LinkageError(e.getMessage(), e);
```

Comment:
Could you revert this too?

Reply:
oop, sure

Reply:
@hellosagar Looks like you forget to revert it.

Reply:
my bad 🤦‍♂️, fixed it